.github/workflows/publish.yml

```diff
@@ -25,4 +25,8 @@ jobs:
       with:
         java-version: 11
     - name: Start deployment
+      env:
+        CI_DEPLOY_USERNAME: ${{ secrets.CI_DEPLOY_USERNAME }}
+        CI_DEPLOY_PASSWORD: ${{ secrets.CI_DEPLOY_PASSWORD }}
+        SPOONBOT_PARAPHRASE: ${{ secrets.SPOONBOT_PARAPHRASE }}
```

Comment:
These are the secrets I will send you via the encrypted channel and then we shall try if we are able to deploy.

Reply:
ack

Reply:
@monperrus Could you send me your GPG ID for that?